### ISSUE TITLE
Syncs are taking hours after upgrading instance type

### ISSUE THREAD

**Issue by kyle-cheung** [airbytehq/airbyte#7656]
## Enviroment
- **Airbyte version**:  0.30.29-alpha
- **OS Version / Instance**: Amazon Linux 2 / AWS EC2 t3.xlarge
- **Deployment**: Docker
- **Source Connector and version**: postgres 0.3.13, salesforce 0.2.5, zendesk support 0.1.4
- **Destination Connector and version**: Snowflake 0.3.16
- **Severity**: Critical
- **Step where error happened**: Sync job
## Current Behavior
Just got off a call with @marcosmarxm trying to debug my issue. Earlier this week I had to reset my connection because I was adding in a new table from my source. At this time my instance was AWS EC2 t2.large. It reached 100% CPU utilization and the machine crashed, it ended up taking around 12 hours to complete when it normally takes 20 minutes.

After that incident, today (3 days later), I had to reset my connection again for a new table and the asme thing happened, 100% CPU utilization. So earlier today around 11am, we upgraded our machine to AWS EC2 t3.xlarge. And now normal syncs (not a reset) is taking exponentially longer to complete. My salesforce connection which normally takes 5 minutes has been running for over 3 hours. My Zendesk sync normally takes 3/4 minutes is now taking 25 minutes to complete. The normalization schedule is not appearing in the logs either. 

At first we thought Snowflake might be rejecting the connection but after being stuck 5 minutes at destination thread it somehow went through. But every step is taking a lot longer than before i upgraded my instance type.

## Expected Behavior
Airbyte should be functioning as normal and syncing my tables in a reasonable amount of time. 

## Logs
Log 3152 is a zendesk connection operating as usual
[logs-3152-0.txt](https://github.com/airbytehq/airbyte/files/7479258/logs-3152-0.txt)

Log 3201 is the log for the zendesk sync that took 25 minutes
[logs-3201-0 (1).txt](https://github.com/airbytehq/airbyte/files/7479261/logs-3201-0.1.txt)

## Steps to Reproduce
1. Upgrade instance from t2.large to t3.xlarge
2. Click sync now

## Are you willing to submit a PR?
Not technical enough

**Comment by kyle-cheung** [comment]
cc @marcosmarxm

**Comment by kyle-cheung** [comment]
the logs also aren't being output properly

**Comment by kyle-cheung** [comment]
today it is back to snowflake rejecting the connection
```
source - 2021-11-05 15:14:03 ERROR () LineGobbler(voidCall):82 - INFO: I/O exception (net.snowflake.client.jdbc.internal.apache.http.NoHttpResponseException) caught when processing request to {s}->https://snowflakecomputing.com:443: The target server failed to respond
source - 2021-11-05 15:14:03 ERROR () LineGobbler(voidCall):82 - Nov 05, 2021 3:14:03 PM net.snowflake.client.jdbc.internal.apache.http.impl.execchain.RetryExec execute
source - 2021-11-05 15:14:03 ERROR () LineGobbler(voidCall):82 - INFO: Retrying request to {s}->https:///snowflakecomputing.com:443
```

**Comment by kyle-cheung** [comment]
I updated the loading method from S3 staging to standard insert it works

**Comment by kyle-cheung** [comment]
Normalization is now taking over 20 minutes with standard inserts

**Comment by marcosmarxm** [comment]
@kyle-cheung this was solved after 0.30.38-alpha?

**Comment by ChristopheDuong** [comment]
Incremental normalization with Snowflake will get a speed-up here https://github.com/airbytehq/airbyte/pull/8088

**Comment by kyle-cheung** [comment]
resolved with 0.32.8-alpha
